Stripe/STPPaymentContext.m

```diff
@@ -687,9 +688,45 @@ - (void)requestPayment {
     }];
 }
 
+/**
+ @param completion The completion block that gets passed to the paymentContext:didCreatePaymentResult
+ method on self.delegate.
```

Comment:
I think this method deserves better documentation, and maybe a better name.

What does it do to handle the source? (attaches the source to the customer, creates a payment result, passes it to the delegate)

When does the `completion` block get called? (if there's an error attaching the source, or once the `STPPaymentContextDelegate` has finished handling the `STPPaymentResult`, probably by telling their backed to charge it)